.github/workflows/realm-generator.yml

```diff
@@ -0,0 +1,64 @@
+name: Realm Generator
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+  pull_request:
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+jobs:
+  Generator_CI:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest] # windows is not tested ue to bug in how error messages are rendered
+        channel: [stable, beta]
```

Comment:
I suggest testing this on stable only. Later we could handle `beta`  in some scheduled nightly runs

Reply:
👌

Reply:
Fixed